src/proto/message_translation/er_force_world.cpp

```diff
@@ -9,14 +9,15 @@ Ball createBall(world::SimBall sim_ball, Timestamp timestamp)
     return ball;
 }
 
+// correct issue #2328: avoid division by 0
 Robot createRobot(world::SimRobot sim_robot, Timestamp timestamp)
 {
     const RobotId id(sim_robot.id());
     const Point position(sim_robot.p_x(), sim_robot.p_y());
     const Vector velocity(sim_robot.v_x(), sim_robot.v_y());
 
     const RobotState state(position, velocity, Angle::fromRadians(sim_robot.angle()),
-                           Angle::atan(sim_robot.r_y() / sim_robot.r_x()));
+                           Angle::zero());
```

Comment:
I think this might be the right way to do it:
```suggestion
                          AngularVelocity::fromRadians(sim_robot.r_z()));
```